src/lib/mina/local-blockchain.ts

```diff
@@ -63,18 +81,14 @@ function LocalBlockchain({
     }
   }
 
-  let testAccounts: {
-    publicKey: PublicKey;
-    privateKey: PrivateKey;
-  }[] = [];
+  let testAccounts = [] as never as TupleN<TestPublicKey, 100>;
 
-  for (let i = 0; i < 10; ++i) {
+  for (let i = 0; i < 100; ++i) {
```

Comment:
@harrysolovay I just realized that this takes 2ms per public key (it does an elliptic curve scaling), so 200ms for 100 accounts feels a bit excessive to do every time we instantiate local blockchain.

Maybe we can create these test accounts lazily and still preserve the same API with some trickery. If not, I'd prefer to create only 20 or so.

All of this can be a separate PR